lib/galaxy/webapps/galaxy/api/quotas.py

```diff
@@ -15,16 +19,154 @@
 )
 from galaxy.actions.admin import AdminActions
 from galaxy.exceptions import ActionInputError
+from galaxy.managers.context import ProvidesUserContext
+from galaxy.managers.quotas import QuotasManager
+from galaxy.quota._schema import (
+    CreateQuotaPayload,
+    UpdateQuotaPayload,
+)
+from galaxy.schema.fields import EncodedDatabaseIdField
 from galaxy.web.params import QuotaParamParser
 from galaxy.webapps.base.controller import (
     BaseAPIController,
     url_for,
     UsesQuotaMixin
 )
+from . import (
+    Depends,
+    get_admin_user,
+    get_trans,
+)
 
 log = logging.getLogger(__name__)
 
 
+# TODO: This FastAPI router is disabled. Please rename it to `router` when the database session issues are fixed.
+_router = APIRouter(tags=['quotas'])
```

Comment:
Want to rebase this to actually use this now that it works?

Reply:
I am afraid the TODO comment here is misleading, is not disabled because of the session issues 😅 
The real reason to be disabled is that the manager classes are not implemented yet, [see this](https://github.com/galaxyproject/galaxy/pull/11315/files#diff-d0ec4c74ff591699ce3812a6a6cbd036906c45c4b3b3616f36915fd1e1eb3e5cR20).

I was waiting for some advice on how to proceed, [see here](https://github.com/galaxyproject/galaxy/pull/11315#issuecomment-776674134) :)

Reply:
I'm going to regret saying it - but I don't know that you have to wait on  #10977. Probably better to merge this sooner. It is an easier thing to do.

Reply:
Ok, thanks @jmchilton, I will mark this as a draft again and continue working on it as soon as I can :)